uktrade/data-hub-frontend
# Reactify OMIS receipt page

## Description of change

The OMIS payment receipt page has been reactified. The [react-to-print](https://github.com/gregnb/react-to-print) dependency has been added to allow the user to print the receipt.

## Test instructions

Go to an order where the status is `paid` or `complete`. It should be possible to view the payment receipt. It should also be possible to print the receipt page.

## Screenshots

### Before

_Add a screenshot_

### After

_Add a screenshot_

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
